core/ir/instr_api.h

```diff
@@ -299,10 +299,15 @@ struct _instr_t {
     uint opcode;
 
     union {
-        /* PR 251479: offset into instr's raw bytes of rip-relative 4-byte displacement */
+#    ifdef X86
+        /* PR 251479: offset into instr's raw bytes of rip-relative 4-byte displacement.
+         * This field is valid when instr_t isa_mode is DR_ISA_X86.
+         */
         byte rip_rel_pos;
+#    endif
         /* PR #6691: size of source data (i.e., read) an instruction operates on.
```

Comment:
Probably no need to cite the PR (I know the one above does...prob shouldn't be there either; that's what git blame is for).

Reply:
Yeah, it is a bit weird, there are a few comments (probably old ones) that cite the PR, but I guess we don't do that anymore.
Removed both PR citations.